tools/src/main/scala/caliban/tools/SchemaWriter.scala

```diff
@@ -63,7 +63,7 @@ object SchemaWriter {
       "%s:%s ZStream[Any, Nothing, %s]".format(
         safeName(field.name),
         if (field.args.nonEmpty) s" ${argsName(field, od)} =>" else "",
-        writeType(field.ofType)
+        writeMaybeEffectType(od, field)
```

Comment:
I think this one should not be lazy since it's already inside `ZStream`

Reply:
Good point, will fix it

Reply:
done